src/par/src/autocluster.cpp

```diff
@@ -1953,9 +1953,9 @@ void AutoClusterMgr::partitionDesign(unsigned int max_num_macro,
       if (map_iter->second->getNumMacro() > 0) {
         vector<Instance*> macro_vec = map_iter->second->getMacros();
         for (int i = 0; i < macro_vec.size(); i++) {
-          Cell* cell = network_->cell(macro_vec[i]);
-          const char* cell_name = network_->name(cell);
-          dbMaster* master = db_->findMaster(cell_name);
+	      const char* inst_name = network_->pathName(macro_vec[i]);
+	      dbInst* inst = block_->findInst(inst_name);
+	      dbMaster* master = inst->getMaster();
```

Comment:
Just use network_->staToDb(cell) to get the dbMaster instead of the name

Reply:
The situation here is that sta network we are using is out of synch with the odb at this stage as the macro masters have swapped with the wrapped master. The instnames are still the same. So I was using the instname from sta network to find the corresponding inst from odb and and the corresponding master  to determine the width and height of the macro.  (Previously it was using the master from sta network, which was the unwrapped macro and had reduced dimensions, which was why some of 10 um halos were showing up as 9 um halos in the GUI)

Reply:
Ok that makes sense.  Just another reason we need to wean off of wrapping.